docs/ref/forms/validation.txt

```diff
@@ -205,6 +205,15 @@ with ``code``\s and ``params`` but a list of strings will also work::
         _('Error 2'),
     ])
 
+If you detect multiple errors in multiple fields in your form, for instance
```

Comment:
It'll also work in `Model.clean()` obviously but these are the forms docs, which is a bit of a problem, `Model.clean()` is actually the only place where this pattern is recommended, `Form.clean()` should use `Form.add_error()` instead.